apis_configs/fence_settings.py

```diff
@@ -68,6 +68,17 @@
         ENABLED_IDENTITY_PROVIDERS = data['ENABLED_IDENTITY_PROVIDERS']
         APP_NAME = data['APP_NAME']
         HTTP_PROXY = data['HTTP_PROXY']
+        os.environ["GOOGLE_API_KEY"] = data['GOOGLE_API_KEY']
+        os.environ["GOOGLE_PROJECT_ID"] = data['GOOGLE_PROJECT_ID']
+        os.environ["GOOGLE_ADMIN_EMAIL"] = data['GOOGLE_ADMIN_EMAIL']
+        os.environ["GOOGLE_IDENTITY_DOMAIN"] = data['GOOGLE_IDENTITY_DOMAIN']
+        os.environ["GOOGLE_CLOUD_IDENTITY_ADMIN_EMAIL"] = (
+            data['GOOGLE_CLOUD_IDENTITY_ADMIN_EMAIL']
+        )
+
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = (
+    "/var/www/fence/google_secret.json"
+)
 
 DEFAULT_LOGIN_URL_REDIRECT_PARAM = 'redirect'
```

Comment:
Should we replace `data[X]` with `data.get(X)`, so it doesn't explode if the json configuration is missing some keys.  Even better - fall back to reasonable default values, and maybe log a message ...?

Reply:
yeah, that's probably a better idea

Reply:
fixed in 8347fc3